Beocreate2/beo-extensions/ui-settings/menu.html

```diff
@@ -81,5 +90,11 @@ <h2>Navigation shortcuts</h2>
 		</div>
 	</div>
 </div>
+<div id="myNav" class="overlay"  data-stylesheet="screensaver.css">
+
+	<div id='stars' class="st" ></div>
+	<div id='stars2' class="st"></div>
+	<div id='stars3' class="st"></div>
 
+</div>
```

Comment:
This is the HTML overlay that shows when the timeout fires.  It is a parallax black and white starfield animated from the css in Beocreate2/beo-extensions/ui-settings/ui-settings.css. I will clean up the id/class names to be more pertinent.